Libraries/Image/RCTResizeMode.h

```diff
@@ -13,6 +13,7 @@ typedef NS_ENUM(NSInteger, RCTResizeMode) {
   RCTResizeModeCover = UIViewContentModeScaleAspectFill,
   RCTResizeModeContain = UIViewContentModeScaleAspectFit,
   RCTResizeModeStretch = UIViewContentModeScaleToFill,
+  RCTResizeModeRepeat = 99, // Pick a high number to make sure we don't conflict with iOS enum values.
```

Comment:
Since this is an NSInteger, maybe use `-1` to add our own custom entries? We could just keep them incremental, since we control how this gets mapped to/from JS anyway.